resources/server/messages/middleware/emitMessage.ts

```diff
@@ -4,6 +4,11 @@ import MessagesService from '../messages.service';
 const exp = global.exports;
 
 // FIXME: Fix this, oh wait, no one contributes anymore
-exp('emitMessage', async ({ senderNumber, targetNumber, message }: EmitMessageExportCtx) => {
-  await MessagesService.handleEmitMessage({ senderNumber, targetNumber, message });
+exp('emitMessage', async ({ senderNumber, targetNumber, message, embed }: EmitMessageExportCtx) => {
+  await MessagesService.handleEmitMessage({
+    senderNumber,
+    targetNumber,
+    message,
+    embed: embed ?? JSON.stringify(embed),
```

Comment:
Would it not be more appropriate to just return an empty string if we don't have an embed? Instead of serializing "nothing".

Reply:
My bad, I meant to use `&&` and not `??`.